packages/atomic/src/components.d.ts

```diff
@@ -205,7 +205,7 @@ export namespace Components {
         /**
           * The Headless Engine.
          */
-        "engine": Engine;
+        "engine": Engine<object>;
```

Comment:
I don't understand why the object generic type was added. I see in other parts of the code you mention Engine<FieldsSection> but I thought the Engine was a global mechanism capable of handling all potential reducers.

Reply:
Yep, the engine is still able to handle all kinds of reducers. The Engine "type" by default though is set to `SearchAppState`, meaning all possible reducers are registered. 

While this is the case when configuring the engine with `reducers: searchAppReducers`, it is not the case when passing `reducers: {},`. Typescript detected this, and showed an error. I specified the generic to tell it that it should not expect the full search app state.